src/app/permissions/permissions.component.ts

```diff
@@ -0,0 +1,90 @@
+import { ChangeDetectorRef, Component, Input, OnInit } from '@angular/core';
+import { User } from '../users/user';
+import { finalize } from 'rxjs/operators';
+import { UsersService } from '../users/users.service';
+import { UserPermissionScope } from '../users/user-permission-scope';
+import { HttpErrorResponse } from '@angular/common/http';
+import { GlobalErrorHandlerService } from '../error/global-error-handler.service';
+import { UserPermissionRequest } from './user-permission-request';
+import { UserPermission } from '../users/user-permission';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'app-permissions',
+  templateUrl: './permissions.component.html',
+  styleUrls: ['./permissions.component.scss']
+})
+export class PermissionsComponent implements OnInit {
+
+  errorMessage: string = null;
+  authorizedSkillsUsers: User[] = [];
+  authorizedProfileUsers: User[] = [];
+
+  allowAll = new FormControl();
+
+  @Input() savingInProgress: boolean;
+
+  constructor(private usersService: UsersService,
+              private changeDetector: ChangeDetectorRef,
+              private globalErrorHandlerService: GlobalErrorHandlerService) {
+
+  }
+
+  ngOnInit() {
+    this.loadAuthorizedUsers();
+  }
+
+  savePermissions() {
+    this.savingInProgress = true;
+    const userSkillPermission: UserPermissionRequest = {
+      scope: UserPermissionScope.READ_USER_SKILLS,
+      authorizedUserIds: this.authorizedSkillsUsers.map(item => item.id)
+    };
+    const userProfilePermission: UserPermissionRequest = {
+      scope: UserPermissionScope.READ_USER_PROFILE
+    };
+
+    if (this.allowAll.value === true) {
+      userProfilePermission.allUsersAuthorized = true;
```

Comment:
This is still discussed in PR https://github.com/T-Systems-MMS/skoop-server/pull/81

The server API to grant permission to all users may change, so please wait until there is a stable contract.